frontend/src/components/camp/CampPrint.vue

```diff
@@ -5,16 +5,15 @@
       <v-btn color="primary" class="mt-5"
              :href="previewUrl"
              target="_blank">
-        Open print preview
+        {{ $tc('entity.camp.print.openPrintPreview') }}
```

Comment:
"Open print preview" is not a field of a camp like "title" or "motto". So this key should be `components.camp.campPrint.openPrintPreview` (also remember to change it in the JSON)

Reply:
done